hub/compression.py

```diff
@@ -80,12 +86,14 @@
 
 AUDIO_COMPRESSIONS = ["mp3", "flac", "wav"]
 
+READ_COMPRESSIONS = ["palm", "pdf"]
```

Comment:
These should be in image compressions as well.

Reply:
But palm and pdf are read-only file formats.